modules/core/src/v2/coins/algo.ts

```diff
@@ -304,15 +306,20 @@ export class Algo extends BaseCoin {
   }
 
   isStellarSeed(seed: string): boolean {
```

Comment:
Wow, why do we have stellar code in algo? This is going to break when we do the modularization project. Each coin family needs to stand on it's own.

Reply:
Yeah, the main problem here is that the ed29915 custody keys were stored in Stellar format, not the core one, so every time we want to reuse them, we have to extract the core prv/pub and then we can use it.

We explored the idea of getting a new key for HBAR to prevent this, but apparently it is not that easy to create and store such key.

Reply:
Got it - thanks for the background.